src/components/ChatWidget.jsx

```
@@ -144,175 +144,177 @@
+                        className={`mb-3 ${msg.sender === 'user' ? 'text-right' : 'text-left'}`}
+                      >
+                        <div
+                          className={`inline-block max-w-xs px-3 py-2 rounded-lg text-sm ${
+                            msg.sender === 'user'
+                              ? 'bg-[#0076b2] text-white'
+                              : 'bg-gray-100 text-gray-800'
+                          }`}
+                        >
+                          <div className="text-left break-words">
+                            <div 
+                              dangerouslySetInnerHTML={{ 
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)